libs/features/mas/commerce/src/inline-price.js

```diff
@@ -84,7 +84,12 @@ const DISPLAY_TAX_MAP = {
     TEAM_EDU: [GeoMap.sea, GeoMap.kr], // school and uni
 };
 
-/** @type {Commerce.Price.PlaceholderConstructor} */
+/** 
+ * HTML Price Web Component, extends span.
+ * special member _onceSettled_ promise that either resolves or fail with web commerce information.
```

Comment:
it always resolves actually. when it fails, it will add placeholder-failed class to the element.